src/main/java/bank/account/AccountDTO.java

```diff
@@ -0,0 +1,9 @@
+package bank.account;
+
+public record AccountDTO(
+        String id,
+        String iban,
+        int balance,
+        String person_id
```

Comment:
```suggestion
       @JsonProperty("person_id")
        String personId
```
Зміна без підкреслювання, а в самому json - це навпаки, нормально. Тож через JsonProperty перевизначемо як мапиться зміна в json.